cmd/occurrence.go

```diff
@@ -1,7 +1,7 @@
 package cmd
 
 type Occurrence struct {
-	kind    int    // or int? just needs to represent  {debug, log, warn, error}
+	kind    int    // represent  {debug, log, warn, error}
 	id      int    // KubeAuditInfo, ErrorImageTagMissing ...
```

Comment:
Same as above may be using pesudo enum

Reply:
The `id` already an `enum` and defined in `errors.go`